webpack_dev_server.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
```

Comment:
Can you use `set -euf -o pipefail` here? See https://sipb.mit.edu/doc/safe-shell/